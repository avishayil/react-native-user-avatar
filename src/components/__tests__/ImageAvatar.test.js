/* eslint max-len: 0 */

import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';

import {ImageAvatar} from '../';

it('should render <ImageAvatar /> with src, size and imageStyle', () => {
  const tree = renderer.create(<ImageAvatar src="https://dummyimage.com/100x100/000/fff" size={100} imageStyle={{borderRadius: 50, width: 100, height: 100}} />).toJSON();
  expect(tree).toMatchSnapshot();
});

it('should render <ImageAvatar /> with src and size', () => {
  const tree = renderer.create(<ImageAvatar src="https://dummyimage.com/100x100/000/fff" size={100} borderRadius={50} />).toJSON();
  expect(tree).toMatchSnapshot();
});
