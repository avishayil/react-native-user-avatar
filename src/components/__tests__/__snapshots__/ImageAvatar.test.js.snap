// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render <ImageAvatar /> with src and size 1`] = `
<Image
  source={
    Object {
      "uri": "https://dummyimage.com/100x100/000/fff",
    }
  }
  style={
    Array [
      Object {
        "borderRadius": 50,
        "height": 100,
        "width": 100,
      },
      undefined,
    ]
  }
/>
`;

exports[`should render <ImageAvatar /> with src, size and imageStyle 1`] = `
<Image
  source={
    Object {
      "uri": "https://dummyimage.com/100x100/000/fff",
    }
  }
  style={
    Array [
      Object {
        "borderRadius": 50,
        "height": 100,
        "width": 100,
      },
      Object {
        "borderRadius": 50,
        "height": 100,
        "width": 100,
      },
    ]
  }
/>
`;
